content/includes/stories.yaml

```diff
@@ -0,0 +1,161 @@
+  icon_width: 57
+  icon_height: 58
+  icon_mobile_width: 33
+  title@: AMP stories
+  subtitle@: A visual storytelling format for the open web
+  description@: AMP stories immerse readers in tappable, full-screen content. Building on the possibilities of the AMP project, this format enables the creation of visual content that is fast, open, and user-first.
+  cta@: Build an AMP Story
+  url: /docs/tutorials/visual_story
+  background_image: /static/img/stories/background-hero.jpg
+  background_image_width: 1617
+  background_image_height: 1715
+  foreground_image_base: /static/img/stories/hero-phone
+  foreground_image_width: 555
+  foreground_image_height: 933
+  video: /static/video/stories/phone.mp4
+  video_width: 416
+  video_height: 720
+  static_base_image: /static/img/stories/static-phone
+  static_image_height: 365
+  static_image_width: 324
+why:
+  title@: Why AMP Stories?
+  reasons:
+    - title@: Immersive
+      description@: By expanding to visually fill readers’ screens, AMP stories offer creative opportunities for storytelling, with flexibility in design. The tappable experience can be used to create engaging stories.
+      icon: /static/img/stories/icon-immersive.png
+      icon_width: 87
+      icon_height: 119
+      class: immersive
+    - title@: Fast
+      description@: AMP stories are built with the same technology used in AMP pages, meaning they’ll load quickly,  while providing users with a smooth experience.
+      icon: /static/img/stories/icon-fast.png
+      icon_width: 87
+      icon_height: 119
+      class: fast
+    - title@: Open
+      description@: AMP stories are part of the open web and can be shared and embedded across sites and apps without being confined to a single ecosystem.
+      icon: /static/img/stories/icon-open.png
+      icon_width: 87
+      icon_height: 119
+      class: open
+features:
+  title@: AMP story features
+  features:
+    - description@: Supports rich visuals with native video and image performance
+      image: /static/img/stories/features-wired.gif
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+    - description@: Enriches storytelling through animations and tappable interactions
+      image: /static/img/stories/features-people.gif
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+    - description@: Works across mobile and desktop devices on the open web
+      image: /static/img/stories/features-hearst.gif
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+    - description@: Builds on top of AMP’s component library to support features like analytics
+      image: /static/img/stories/features-amp.png
+      image_width: 600
+      image_height: 581
+      image_alt@: A Pod Races Through the Hyperloop for the First Time
+news:
```

Comment:
After talking with Matt, these "quote boxes" should not be clickable.  Please remove URLability. These are quotes that should not go to case studies.  1) Because case study for wapo is old 2) there is no case study for Time.

Reply:
Removed links